rag_api/src/deh/settings.py

```diff
@@ -15,5 +15,7 @@
     "EMBEDDING_MODEL", "sentence-transformers/all-mpnet-base-v2"
 )
 
+SIMILIARITY_THRESHOLD = float(os.environ.get("SIMILARITY_THRESHOLD", "2.0"))
```

Comment:
it's cosine similarity, maybe like 0.5?
```suggestion
SIMILIARITY_THRESHOLD = float(os.environ.get("SIMILARITY_THRESHOLD", "0.5"))
```

Reply:
Changed thresholds in next PR (though may make more sense as a filter so that contexts are exposed to UI vs. exception).